controller/jobs_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strings"
 
 	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
+	controller "github.com/flynn/flynn/controller/client"
```

Comment:
Additional point: `vim-go` can't follow definitions if the package name is different from its path leaf. By naming it, `vim-go` is able to resolve definitions.